src/System.Configuration.ConfigurationManager/tests/System/Configuration/ApplicationSettingsBaseTests.cs

```diff
@@ -103,12 +107,59 @@ public void GetSetIntProperty_SimpleSettings_Ok(bool isSynchronized)
             Assert.Equal(10, settings.IntProperty);
         }
 
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer)),
+            InlineData(true),
+            InlineData(false)]
+        [ActiveIssue("dotnet/corefx #18832", TargetFrameworkMonikers.NetFramework)]
```

Comment:
I don’t think this should be failing with NotSupported.  I’d expect this to just forward to the desktop implementation.  The linked issue looks like it’s lacking a good root cause analysis.